mungegithub/github/github.go

```diff
@@ -1216,6 +1218,27 @@ func (obj *MungeObject) AssignPR(owner string) error {
 	return nil
 }
 
+// CloseIssuef will close the given issue with a message
+func (obj *MungeObject) CloseIssuef(format string, args ...interface{}) error {
+	config := obj.config
+	msg := fmt.Sprintf(format, args...)
+	if err := obj.WriteComment(msg); err != nil {
+		return fmt.Errorf("failed to write comment to %v: %q: %v", *obj.Issue.Number, msg, err)
```

Comment:
For 'open' we put the retry logic down here. It's just taste I guess. I haven't gone to look, but who cares if the close fails? We just find it and try again next time?